packages/shared/src/utils/validation/regex-validation.ts

```diff
@@ -1,8 +1,8 @@
 export const regex_checks = {
     address_details: {
         address_city: /^\p{L}[\p{L}\s'.-]{0,99}$/u,
-        address_line_1: /^([a-zA-Z0-9’'.,:;()@#/-]+\s)*[a-zA-Z0-9’'.,:;()@#/-]{1,70}$/,
-        address_line_2: /^([a-zA-Z0-9’'.,:;()@#/-]+\s)*[a-zA-Z0-9’'.,:;()@#/-]{0,70}$/,
+        address_line_1: /^[\p{L}\p{Nd}\s'.,:;()\u00b0@#/-]{1,70}$/u,
```

Comment:
Any letter of any language, decimals, spaces, a bunch of permitted characters, among them `\u00b0` which stands for `°`, string length to be between 1 and 70.